docs/api/zilliqa/getnodetype.doc.md

```diff
@@ -14,9 +14,8 @@ NotDocumented
 
 Returns node type. The possible return values are:
 
-- `"Not in network, synced till epoch [epoch number]"` if the server has not joined the network and is synced until a specific epoch.
-- `"Seed"` if the server is in lookup node mode and is an archival lookup node.
-- `"Lookup"` if the server is in lookup node mode
+- `"Leader"` if we are currently the lead node
+- `"Validator"` otherwise
```

Comment:
I think we should return (a subset of) the same values that ZQ1 returned. Clients will be expecting to receive one of the existing values, so I'm cautious about inventing new ones. @chetan-zilliqa Can you advise which ZQ1 value is most appropriate for ZQ2 nodes?

Reply:
If I try it on api.zilliqa.com I get:

````
curl -s --request POST \
  --url https://api.zilliqa.com/ \
  --header 'Content-Type: application/json' \
  --data '{
    "id": "1",
    "jsonrpc": "2.0",
    "method": "GetNodeType",
        "params": [""]
}' | jq
{
  "id": "1",
  "jsonrpc": "2.0",
  "result": "Seed"
}
````

So should I just hardcode it to always return Seed then? And update the documentation to say that it always returns seed for backwards compatibility reasons?

Reply:
Only two values would make sense here IMO.
seed - If a node is an API node.
validator - For all other nodes.

Reply:
So it should be okay to hardcode Seed then, right? because GetNodeType doesn't take arguments, it always applies to the node answering the request, and that's guaranteed to be an API node precisely because it is answering the request?

Reply:
This is now updated to always return Seed

Reply:
Makes sense to me :+1: Thanks